src/app/models/RestaurantAddRequest.ts

```diff
@@ -0,0 +1,10 @@
+export class RestaurantAddRequest{
```

Comment:
Remove if not used. Reuse address details since now you know how to use it. Let your backend partner know